src/test/java/org/swisspush/redisques/RedisQuesTest.java

```diff
@@ -40,7 +38,7 @@ public void deployRedisques(TestContext context) {
                 .processorAddress(PROCESSOR_ADDRESS)
                 .redisEncoding("ISO-8859-1")
                 .refreshPeriod(2)
-                .queueConfigurations(Arrays.asList(new QueueConfiguration()
+                .queueConfigurations(Collections.singletonList(new QueueConfiguration()
```

Comment:
I assume it's properly tested so no other component will ever access that immutable list.

Reply:
It's just a unit test

Reply:
And it is correct to assume that parameters are not going to be mutated. It is a bug if someone modifies what you pass without making it explicit in the doc (or better in the method name).